include/fiction/technology/sidb_charge_state.hpp

```diff
@@ -15,5 +15,58 @@ enum class sidb_charge_state
     NEUTRAL,
     NEGATIVE
 };
+/**
+ * Convert the charge state into int (-1,0,1)
+ *
+ * @param cs sidb charge state
+ * @return int representing the SiDB's charge state
+ */
+[[nodiscard]] int transform_to_sign(const sidb_charge_state& cs) noexcept
+{
+    if (cs == sidb_charge_state::POSITIVE)
+    {
+        return +1;
+    }
+    else if (cs == sidb_charge_state::NEGATIVE)
+    {
+        return -1;
+    }
+
+    else if (cs == sidb_charge_state::NEUTRAL)
+    {
+        return 0;
+    }
+
+    else
+    {
+        return 0;
+    }
+
+}
+
+
+[[nodiscard]] sidb_charge_state sign_to_label(const int& sg) noexcept
```

Comment:
No need to pass `int` by reference. An `int` is 32 or 64 bit on most platforms. Its address is too.